study-builder/studies/basil/study.conf

```diff
@@ -29,6 +29,8 @@
 
     "studyDetails": [],
 
+    "studySupportedLanguages": [ "en" ],
```

Comment:
Can you also update `studies/study-schema.conf` file and put some notes/comments describing this new field?

Reply:
Sure, done.